app/models/event_type.rb

```diff
@@ -10,6 +10,7 @@ class EventType < ActiveRecord::Base
 
   alias_attribute :name, :title
 
+  # If LENGTH_STEP > 60 the public schedule must be changed
```

Comment:
This is a constant. Whoever changes this must deal with the outcome. They have been warned. All is good..